src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_params.py

```diff
@@ -163,6 +163,7 @@ def get_hostname_completion_list(prefix, action, parsed_args, **kwargs):  # pyli
 register_cli_argument('webapp log config', 'failed_request_tracing', help='configure failed request tracing', **enum_choice_list(two_states_switch))
 register_cli_argument('webapp log config', 'level', help='logging level', **enum_choice_list(['error', 'warning', 'information', 'verbose']))
 register_cli_argument('webapp log config', 'web_server_logging', help='configure Web server logging', **enum_choice_list(['off', 'storage', 'filesystem']))
+register_cli_argument('webapp log config', 'sas_url', help='configure Sas Url, only applicable when --web-server-logging is set to \'storage\'')
```

Comment:
Should it be 'SAS' in the help doc? All capital as it is an abbreviation.

Reply:
The storage SAS is not a URL. It is a part of the query string composed of multiple key-value pairs. It is attached to the storage container/share URL as a part of the query string for access.

If this is actually a storage URL. Is it possible that SAS token is not required if the storage account's permission is set to public? Though it is not a common nor recommended practice, this makes the parameter name and help doc misleading. I suggest changing the parameter to 'storage_url'.

Reply:
I see what you're saying.. but keeping it as SAS will keep is consistent with the model defined in the sdk and what we see in the return object.

Reply:
Command line users won't see the SDK, so I don't necessarily worry about that. Is the property in the return object, aka the JSON string printed to stdout, named `sas_url`? If so, I would establish the relationship between the model property and the parameter name in help documentation.

After all, we're writing a user-oriented tool. The parameters shouldn't be bound to the SDK names.

I won't block this PR for this issue. But I do think the parameter can be named better.

Reply:
By the way, once parameter name is introduced, it becomes very difficult to retire.